packages/common/test/util/references.test.js

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { expandReferences, normalizeOauthConfig } from '../src/util';
+import { expandReferences, normalizeOauthConfig } from '../../src/util/index.js';
```

Comment:
TODO why does this have `.js`? Does it need it? If so shouldn't everything need `.js`?